pkgs/development/interpreters/luajit/default.nix

```
@@ -10,6 +11,7 @@
 , enableFFI ? true
 , enableJIT ? true
 , enableJITDebugModule ? enableJIT
+, enableGC64 ? false
```

```suggestion
, enableGC64 ? stdenv.hostPlatform.isAarch64
```
If I read [upstream docs](https://github.com/LuaJIT/LuaJIT/commit/bd00094c3b50#diff-f49570660ffcc1927a88797c0f9778cdL212) right, we would otherwise be "downgrading" the default on aarch64.  (we can most likely just disregard mips here)

---

Nice catch, I think only mode for aarch64 is GC64, but still I will apply the change.

---

I didn't investigate much.  It successfully builds even with `false` :shrug: